src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -836,6 +844,13 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+		/**
+		 * Force the tab navigation to display even if there is only one tab
+		 */
+		forceTabs: {
```

Comment:
Maybe `force-navigation` would be more clear?

Reply:
https://github.com/nextcloud-libraries/nextcloud-vue/pull/5740#discussion_r1656805220

👀

Reply:
oh makes sense I was confused by the doc of the prop :sweat_smile:

Reply:
🏓 🏐 🏓